vlinder/src/app/login/page.tsx

```diff
@@ -5,30 +5,13 @@ import { Button, Input } from "@nextui-org/react";
 export default function LoginPage() {
   const widthClass = "w-[70%]"; 
   return (
-    <div className="flex items-center justify-center min-h-screen">
```

Comment:
why is this replaced with shitty form inputs?

Reply:
shity as in no layout

Reply:
as it is functional